bff/test/Duende.Bff.Tests/IAccessTokenRetriever_Extensibility_tests.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Duende Software. All rights reserved.
+// See LICENSE in the project root for license information.
+
+using System;
+using Duende.Bff.Tests.TestHosts;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Xunit.Abstractions;
+using Microsoft.Extensions.Logging.Abstractions;
+using Shouldly;
+
+namespace Duende.Bff.Tests;
+
+/// <summary>
+/// These tests prove that you can use a custom IAccessTokenRetriever and that the context is populated correctly. 
+/// </summary>
+public class IAccessTokenRetriever_Extensibility_tests : BffIntegrationTestBase
```

Comment:
These tests prove that the metadata is set correctly, which is needed when creating a custom IAccessTokenRetriever